src/app/planner/components/room-planner/room-planner.component.ts

```diff
@@ -31,6 +29,7 @@ export class RoomPlannerComponent implements OnInit {
     @Output() drop = new EventEmitter();
     @Output() eventMouseOver = new EventEmitter();
     @Input() initialStartDate: Date;
+    @Input() sessions: any[];
```

Comment:
Just noticed that this <any> looks like a potential future issue, we should think about making it SessionViewModel or similar soon.